developer/src/kmc/src/commands/build.ts

```diff
@@ -102,10 +102,11 @@ async function buildFile(filenames: string[], _options: any, commander: any)  {
   }
 
   for(let filename of filenames) {
-    if(!await build(filename, commanderOptions.outFile, callbacks, options)) {
+    if(!await build(filename, commanderOptions.outFile, callbacks, options) || callbacks.hasFailureMessage()) {
       // Once a file fails to build, we bail on subsequent builds
       await exitProcess(1);
     }
+    kmcAssert(!callbacks.hasFailureMessage(), () => `errors: ${CompilerError.formatEvent(callbacks.messages)}`);
```

Comment:
This can never fire, because of the test on line 105.

Reply:
I don't think we need this because the `build()` function already deals with this.

```suggestion
```